docgen/extras/firebase-admin.database.md

```diff
@@ -0,0 +1,12 @@
+## External API Re-exports
+
+Following externally defined APIs are re-exported from this module entry point for convenience.
```

Comment:
Suggest "The following" here and for firestore.md.

Reply:
Done